conf/profiles/singularity.config

```diff
@@ -2,6 +2,9 @@ process {
     withLabel:ips6_container {
         container = 'docker://interpro/interproscan:6.0.0-beta'
     }
+    withLabel:interpro_n_container {
+        container = 'interpro/interpro-n:latest'
+    }
     withLabel:mobidblite_container {
         container = 'docker://matblum/idrpred:1.0.3'
     }
```

Comment:
We specified specific releases for our other containers, should we do the same for the InterPro-N container?